src/components/code/code.tsx

```diff
@@ -4,7 +4,7 @@ import './Code.scss'
 interface IProps {
     code: boolean
     setCode: any
-    stateComponent: boolean
+    stateComponent: any
```

Comment:
Avoid using 'any'

Reply:
I put any just in order to get code work properly, now I chahged it to boolean and it works just fine.